api/napi/src/interpreter/value.rs

```diff
@@ -105,12 +108,39 @@ pub fn to_value(env: &Env, unknown: JsUnknown, typ: Type) -> Result<Value> {
             Ok(Value::Bool(js_bool.get_value()?))
         }
         Type::Color => {
-            let js_color: JsExternal = unknown.coerce_to_object()?.get("color")?.unwrap();
-            Ok(Value::Brush(Brush::from(env.get_value_external::<Color>(&js_color)?.clone())))
+            if unknown.get_type()?.eq(&ValueType::String) {
```

Comment:
Are you sure about the `?` here? It seems to me that if that fails, we're not even going to try the "native" color code path in the else. Shouldn't we?

I think a more robust overall logic would be this:

1. Try to coerce the unknown to an object, extract the color property, attempt to extract the `Color` external.
2. If any of the above steps fail, attempt to coerce to a string and then try to the CSS parsing.

The same applies to the brush code. Perhaps there's a way to avoid the code duplication by sharing the code in a helper function?

Reply:
done